### PR TITLE
Docs/scoping

### PR DESCRIPTION
**Pull request by vijaypemmaraju**
Also contains changes from https://github.com/GrooveLabs/grooveEngine/pull/7731

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -160,6 +160,21 @@ Exceeding the limit will yield an empty response with status code `429`.
 **Note: We continue to count any API calls made while the limit is exceeded.**
 
 
+# Scoping
+
+Scopes are defined at an OAuth App level. The below table describes the scopes and the corresponding GraphQL queries/mutation that they allow.
+
+Scope | Queries | Mutations
+----- | ------- | ---------
+`user::flow:read` | `flow`, `flows`, `flows.steps`, `findAddableFlowsForPerson` |
```
Comment (jaebradley) [inline source/index.html.md]: Unrelated to this per se (but obviously sparked by this documentation)...should we have `flow` / `flows` queries? I thought they were built for mostly PoC purposes?

Related to this documentation - accessing the `steps` field of a flow doesn't seem to me to be a separate "query". You're still executing the `flows` query type, right?

Comment (jaebradley) [inline source/index.html.md]: Maybe a description would be useful? I know Slack has a page dedicated to what the scopes mean. 

image

Also, maybe _some_ background on the general structure of a scope string (what each part means, for example) might be useful to integrators.

**Review by jaebradley — APPROVED** [review]
Couple minor things - overall, looks great!

scope
doc

File: source/index.html.md (hunk 1 above)
Comment (vijaypemmaraju) [reply source/index.html.md]: yeah i think you're right

**Pull request merged**